ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/sdn/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-sdn-provision-chain.metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/sdn/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-sdn-provision-chain.yaml",
```

Comment:
How about move `sdn` folder to `aws/ipi/disconnected/private`, then we can have `cucushift/installer/rehearse/aws/ipi/disconnected/private/sdn/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn/-rovision-chain.yaml`, then we can reference the provision chain in the rhel workflow.

Reply:
Good idea. 
Sure, I can have a try.